velox/exec/tests/PlanNodeToStringTest.cpp

```diff
@@ -13,67 +13,336 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#include "velox/exec/tests/utils/OperatorTestBase.h"
+#include <gtest/gtest.h>
 #include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/functions/prestosql/registration/RegistrationFunctions.h"
+#include "velox/parse/TypeResolver.h"
+#include "velox/vector/tests/VectorTestBase.h"
 
 using namespace facebook::velox;
-using namespace facebook::velox::exec::test;
-class PlanNodeToStringTest : public OperatorTestBase {
+using facebook::velox::exec::test::PlanBuilder;
+
+class PlanNodeToStringTest : public testing::Test, public test::VectorTestBase {
  public:
   PlanNodeToStringTest() {
-    auto rowType = ROW({"c0", "c1"}, {INTEGER(), INTEGER()});
-
-    auto vector = makeRowVector(
-        {makeFlatVector<int32_t>(100, [](auto row) { return row; }),
-         makeFlatVector<int32_t>(100, [](auto row) { return row + 1; })});
-    vectors_.push_back(vector);
+    functions::prestosql::registerAllScalarFunctions();
+    parse::registerTypeResolver();
+    data_ = makeRowVector({
+        makeFlatVector<int16_t>({0, 1, 2, 3, 4}),
+        makeFlatVector<int32_t>({0, 1, 2, 3, 4}),
+        makeFlatVector<int64_t>({0, 1, 2, 3, 4}),
```

Comment:
The third row (`c2`) is not used?

Reply:
Used in TEST_F(PlanNodeToStringTest, aggregation)

```
  auto plan = PlanBuilder()
                  .values({data_})
                  .partialAggregation(
                      {}, {"sum(c0) AS a", "avg(c1) AS b", "min(c2) AS c"})
```